client/src/components/Register/Introduction.scss

```diff
@@ -0,0 +1,53 @@
+$feature-list-content-margin: 20px;
+$feature-list-icon-size: 50px;
+$feature-list-action-list-spacing: 45px;
+
+$font-feature-list-title-size: 16px;
+$font-feature-list-title-weight: bold;
+
+.mfa-feature-list-title {
+  color: $color-grey-text;
+  font-weight: $font-feature-list-title-weight;
+  font-size: $font-feature-list-title-size;
+  margin-bottom: $feature-list-content-margin;
+}
+
+.mfa-feature-list {
+  list-style: none;
+  margin: 0 0 $feature-list-action-list-spacing;
+  padding: 0;
+}
```

Comment:
You could combine `.mfa-feature-list`, `.mfa-feature-list-title` and `.mfa-feature-list-item` with SCSS `&-list {}` etc if you wanted to. We've done that in other parts of the module already. I'd be fine if you don't want to though, because I can see they're semantically separate elements, but consistency is also good.

Reply:
Doesn't BEM prefer that something like a title is defined as an "element" (ie `.mfa-feature-list__title`)?

Reply:
I tend to limit my use of `&` to the Es and Ms of a B - to me, it's easier to parse that way.